ee/api/test/test_property_definition.py

```diff
@@ -72,6 +75,11 @@ def test_search_property_definition(self):
         response_data = response.json()
         self.assertEqual(len(response_data["results"]), 0)
 
+        response = self.client.get(f"/api/projects/@current/property_definitions/?search=set")
```

Comment:
small nit: easy to bork this test if the search query is different to the property definition you added.

An extra nice test: Get all property definitions (without search param) & confirm that count = 2 ($set is excluded)